api/client/network/create.go

```diff
@@ -55,6 +56,7 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.labels, "label", []string{}, "Set metadata on a network")
 	flags.BoolVar(&opts.internal, "internal", false, "Restrict external access to the network")
 	flags.BoolVar(&opts.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(&opts.manuallyAttachable, "manually-attachable", false, "Enable manual container attachment")
```

Comment:
But can't someone on a compromised worker node just create a bunch of services anyway?
Or `docker run --net=container:<some task>`?
Why would a compromised node that can attach containers to any given network be any different than a compromised node that can attach services to any given network?